spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -734,6 +734,22 @@ class CometSparkSessionExtensions
       } else {
         var newPlan = transform(plan)
 
+        // if the plan cannot be run natively then explain why (when appropriate config is enabled)
```

Comment:
Given that ExecRule is going to be invoked multiple times as planning proceeds, this may get logged multiple times for the same plan. Perhaps that is why we need to keep this configurable.

Reply:
That is one reason, yes. The logging could get verbose.

Reply:
I would assume that this feature is useful in development but less so in production since we have the integration with Spark's explain for that.